src/slsV4BucketPolicies.ts

```diff
@@ -0,0 +1,75 @@
+import {
```

Comment:
I wonder if this is possible (not sure) to do this with serverless:
https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml#deployment-bucket
I guess the problem with that link is that it's for the stack specific deployment bucket vs the global all stacks of all branches. But let me know if you find anything worth thinking about.

Reply:
There's not a way with compose at this point so ignore this.